R/series__series.R

```diff
@@ -265,19 +268,34 @@ Series_struct = method_as_active_binding(\() series_make_sub_ns(self, expr_struc
 #' s2 = pl$Series(1:3, "a", dtype = pl$Float32)
 #' s2
 pl_Series = function(
-    values,
+    ...,
+    values = NULL,
     name = NULL,
     dtype = NULL,
-    ...,
     nan_to_null = FALSE) {
   uw = function(x) unwrap(x, "in pl$Series():")
 
+  if (!missing(...)) {
```

Comment:
> IIUC, `pl$Series(1:3)` should still work as before so this warning should appear only if `!missing(...) && length(...) > 1` or sth equivalent.

I don't want to implement the fallback mechanism of Python Polars for `name` to `values` here, because in Python we can guess the first argument as `name` in the following case, but in R it is not possible to determine whether they intend `name` or `values` impossible.

```python
pl.Series("foo") # <- name, not values
```

```r
pl$Series("foo") # <- name or values????
```

Also, R has `as_polas_series()`, which takes a vector as its first argument, so users can use that.

Reply:
I see, I didn't realize `pl.Series("foo")` only created a named but empty Series in Python. In that case I'm fine with warning anytime there's something in `...`.

Still I think the warning could be clearer and mention `as_polars_series()` since we want to push the usage of our `as_*` functions anyway:

```r
warning(
  "`pl$Series()` will handle unnamed arguments differently as of 0.17.0:\n",
  "- until 0.17.0, the first argument corresponds to the values and the second argument to the name of the Series.\n",
  "- as of 0.17.0, the first argument will correspond to the name and the second argument to the values.\n",
  "Use named arguments in `pl$Series()` or replace `pl$Series(<x>)` by `as_polars_series(<x>)` to silence this warning.\n"
)
```